exercises/minesweeper/minesweeper_test.py

```diff
@@ -1,162 +1,71 @@
-                     "   "]
-        out = ["111",
-               "1*1",
-               "111"]
-        self.assertEqual(annotate(minefield), out)
+        minefield = ["   ", " * ", "   "]
+        expected = ["111", "1*1", "111"]
+        self.assertEqual(annotate(minefield), expected)
 
     def test_space_surrounded_by_mines(self):
-        minefield = ["***",
-                     "* *",
-                     "***"]
-        out = ["***",
-               "*8*",
-               "***"]
-        self.assertEqual(annotate(minefield), out)
+        minefield = ["***", "* *", "***"]
+        expected = ["***", "*8*", "***"]
+        self.assertEqual(annotate(minefield), expected)
 
     def test_horizontal_line(self):
         minefield = [" * * "]
-        out = ["1*2*1"]
-        self.assertEqual(annotate(minefield), out)
+        expected = ["1*2*1"]
+        self.assertEqual(annotate(minefield), expected)
 
     def test_horizontal_line_mines_at_edges(self):
         minefield = ["*   *"]
-        out = ["*1 1*"]
-        self.assertEqual(annotate(minefield), out)
+        expected = ["*1 1*"]
+        self.assertEqual(annotate(minefield), expected)
 
     def test_vertical_line(self):
-        minefield = [" ",
-                     "*",
-                     " ",
-                     "*",
-                     " "]
-        out = ["1",
-               "*",
-               "2",
-               "*",
-               "1"]
-        self.assertEqual(annotate(minefield), out)
+        minefield = [" ", "*", " ", "*", " "]
+        expected = ["1", "*", "2", "*", "1"]
+        self.assertEqual(annotate(minefield), expected)
 
     def test_vertical_line_mines_at_edges(self):
-        minefield = ["*",
-                     " ",
-                     " ",
-                     " ",
-                     "*"]
-        out = ["*",
-               "1",
-               " ",
-               "1",
-               "*"]
-        self.assertEqual(annotate(minefield), out)
+        minefield = ["*", " ", " ", " ", "*"]
+        expected = ["*", "1", " ", "1", "*"]
+        self.assertEqual(annotate(minefield), expected)
 
     def test_cross(self):
-        minefield = ["  *  ",
-                     "  *  ",
-                     "*****",
-                     "  *  ",
-                     "  *  "]
-        out = [" 2*2 ",
-               "25*52",
-               "*****",
-               "25*52",
-               " 2*2 "]
-        self.assertEqual(annotate(minefield), out)
-
-    def test_large_annotate(self):
-        minefield = [" *  * ",
-                     "  *   ",
-                     "    * ",
-                     "   * *",
-                     " *  * ",
-                     "      "]
-        out = ["1*22*1",
-               "12*322",
-               " 123*2",
-               "112*4*",
-               "1*22*2",
-               "111111"]
-        self.assertEqual(annotate(minefield), out)
-
-    # Additional test for this track
```

Comment:
Please see the [generator documentation](https://github.com/exercism/python/blob/master/docs/GENERATOR.md#conventions) for info on adding track-specific test cases.